CONTRIBUTING.md

```diff
@@ -43,6 +43,14 @@ Don't forget to add your new article to the `SUMMARY.md` to let it be rendered t
 
 Please make `Draft Pull requests` early so we can follow your progress and can give early feedback (see the following section).
 
+## Style guide
+
+- Follow the book [style guide](https://github.com/rust-lang/book/blob/master/style-guide.md).
+- Follow [RFC 1574](https://github.com/rust-lang/rfcs/blob/master/text/1574-more-api-documentation-conventions.md#appendix-a-full-conventions-text). TL;DR:
+  - Use American English
```

Comment:
```suggestion
  - Use formal American English
```

Reply:
That is near to impossible for us to check as we are international and not English natives. If so I would prefer British English over American, I don't think this is a good rule, to be honest. Let's discuss about it.

Also we want to make it as easy as possible for people to write and not put more and more rules and everything in the way. So I'm really unsure the direction we are taking with this.

Reply:
It doesn't have to be as formal as you would write to Queen Elizabeth, but it should in my opinion also not be slang or too informal.

Reply:
https://rust-unofficial.github.io/patterns/idioms/mem-replace.html#advantages

> Look ma, no allocation! Also you may feel like Indiana Jones while doing it.

I love that actually and I find it kind of what a collaborative book is. Different styles of different authors and not something that is made out of stone and would always sound the same. For sure if it gets too colloquial/informal we can interfere in the PR process, but imho no need to write that down as a rule. I'm trusting the people maintaining this now and in the long-term future that there will be a rational about how a PR/article is written and if it fits to the books style.

Reply:
I also like informal style, see my comment in the relative issue. I will resolve this conversation without committing this suggestion.